### PR TITLE
[FEATURE] enable build with CSS variables

### PR DESCRIPTION
**Pull request by flovogt**

### CODE REVIEW

File: lib/cli/commands/build.js [hunk 1]
```diff
@@ -111,7 +118,9 @@ build.builder = function(cli) {
 			"Build project and dependencies in dev mode, except \"sap.ui.core\" and \"sap.m\" " +
 			"(useful in combination with --include-task)")
 		.example("ui5 build dev",
-			"Build project and dependencies in dev mode. Only a set of essential tasks is executed.");
+			"Build project and dependencies in dev mode. Only a set of essential tasks is executed.")
+		.example("ui5 build --experimental-css-variables",
+			"Build project and dependencies with the usage of CSS variables");
```
Comment (codeworrior) [inline lib/cli/commands/build.js:123]: [wording] "usage" sounds as if the build or the build results at runtime would make use of the CSS variables. But this is not the  case. In essence, we just create additional artefacts that - by default - are not used anywhere.

**Comment by matz3** [comment]
PR should go into the `next` branch, not master.

File: lib/cli/commands/build.js (hunk 1 above)
Comment (flovogt) [reply lib/cli/commands/build.js:123]: Changed the wording in the new PR https://github.com/SAP/ui5-cli/pull/501.

**Comment by flovogt** [comment]
Superseded by https://github.com/SAP/ui5-cli/pull/501.

**Pull request closed without merging**